tools/eslint/plugins/botbuilder.js

```
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const _config = {
+    disableLine: {},
+};
+
+module.exports = plugin = {
```

ESLint

[error] 'plugin' is not defined.

(no-undef)